### DISCUSSION TITLE
Help - GitHub Foundations Exam

### DISCUSSION THREAD

**Discussion by FedeMontero** [community/community#136648] [New to GitHub]
Good morning,

I kindly ask for your advice regarding my GitHub Foundations certification because I don't know how to resolve my issue.

On August 7, 2024, I logged in at the scheduled time (6:00 PM Uruguay Standard Time) to take the certification exam. However, the Secure Browser never progressed beyond the initial screen. Before the exam, I conducted practice tests, checked my computer, and confirmed that everything was functioning properly.

Despite my efforts, I was unable to take the exam.

For the past two weeks, I’ve been calling PSIexams.com. At first, they told me not to worry, but when I called yesterday, they advised me to contact GitHub.

I would greatly appreciate any help, as I’m not sure what to do next.

Thank you very much.

Best regards.

**Comment by ghostinhershell** [comment]
Hi @FedeMontero, the best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them.
